testing/colormake/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer: Henrik Riomar <henrik.riomar@gmail.com>
+pkgname=colormake
+pkgver=0.9.20140503
+pkgrel=0
+pkgdesc="A simple wrapper around make to colorize the output"
+url="http://bre.klaki.net/programs/colormake"
+arch="noarch"
+license="GPL2+"
+depends="perl"
+options="!check"
```

Comment:
I'm not sure if the $PATH part is needed, but yes, running the binary is an okay smoke test. :)